### DISCUSSION TITLE
Been charged twice by GitHub Copilot (yearly package) - am on the monthly atm

### DISCUSSION THREAD

**Discussion by gezibash** [community/community#107448] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hey guys, I've been charged twice by GitHub Copilot for the yearly bill, i.e. $100 * 2, while I'm subscribed to the monthly package. Do you guys think you can take a look? Not sure who to contact about this.

I'm okay with paying the yearly bill, as long as my Copilot subscription reflects that. Don't wanna get charged monthly now.

Also, it would be cool if you can refund me the 2nd $100. I could use that money 😅

**Comment by davecheney** [comment, marked as answer]
@gezibash i'm very sorry this has occurred. We cannot address billing issues via this forum, please contact support directly; https://support.github.com/contact

**Comment by ppcamp** [comment]
Same here 😢

**Reply by davecheney** [reply]
i'm very sorry this has occurred. We cannot address billing issues via this forum, please contact support directly; https://support.github.com/contact
